lib/everypolitician/popolo/event.rb

```diff
@@ -2,7 +2,41 @@ module Everypolitician
   module Popolo
     class Events < Collection; end
     class Event < Entity
-      attr_accessor :start_date, :end_date
+      def initialize(document, _p)
+        @document = document
+      end
+
+      def start_date
+        document[:start_date]
+      end
+
+      def end_date
+        document[:end_date]
+      end
+
+      def id
+        document[:id]
+      end
+
+      def name
+        document[:name]
+      end
+
+      def classification
+        document[:classification]
+      end
+
+      def organization_id
+        document[:organization_id]
+      end
+
+      def wikidata
+        document[:wikidata]
```

Comment:
This method is only tested to return `nil`. I'm curious under what circumstances it might _not_ do so, so it would be useful to see a test that actually returns something here (against live data)